test/unit/component.test.js

```diff
@@ -483,71 +502,71 @@ test('should not retrigger a listener when the listener calls triggerReady', fun
     }
   };
 
-  var comp = new Component(getFakePlayer(), {});
-
   comp.ready(readyListener);
   comp.triggerReady();
 
   this.clock.tick(100);
 
-  equal(timesCalled, 1, 'triggerReady from inside a ready handler does not result in an infinite loop');
+  QUnit.equal(timesCalled, 1,
+    'triggerReady from inside a ready handler does not result in an infinite loop');
 });
 
-test('should add and remove a CSS class', function(){
-  var comp = new Component(getFakePlayer(), {});
+QUnit.test('should add and remove a CSS class', function() {
+  const comp = new Component(getFakePlayer(), {});
 
   comp.addClass('test-class');
-  ok(comp.el().className.indexOf('test-class') !== -1);
+  QUnit.ok(comp.el().className.indexOf('test-class') !== -1);
   comp.removeClass('test-class');
-  ok(comp.el().className.indexOf('test-class') === -1);
+  QUnit.ok(comp.el().className.indexOf('test-class') === -1);
   comp.toggleClass('test-class');
-  ok(comp.el().className.indexOf('test-class') !== -1);
+  QUnit.ok(comp.el().className.indexOf('test-class') !== -1);
   comp.toggleClass('test-class');
-  ok(comp.el().className.indexOf('test-class') === -1);
+  QUnit.ok(comp.el().className.indexOf('test-class') === -1);
 });
 
-test('should show and hide an element', function(){
-  var comp = new Component(getFakePlayer(), {});
+QUnit.test('should show and hide an element', function() {
+  const comp = new Component(getFakePlayer(), {});
 
   comp.hide();
-  ok(comp.hasClass('vjs-hidden') === true);
+  QUnit.ok(comp.hasClass('vjs-hidden') === true);
   comp.show();
-  ok(comp.hasClass('vjs-hidden') === false);
+  QUnit.ok(comp.hasClass('vjs-hidden') === false);
 });
 
-test('dimension() should treat NaN and null as zero', function() {
-  var comp, width, height, newWidth, newHeight;
-  width = 300;
-  height = 150;
+QUnit.test('dimension() should treat NaN and null as zero', function() {
+  let newWidth;
 
-  comp = new Component(getFakePlayer(), {}),
+  const width = 300;
+  const height = 150;
+
+  const comp = new Component(getFakePlayer(), {});
   // set component dimension
 
   comp.dimensions(width, height);
 
   newWidth = comp.dimension('width', null);
 
   notEqual(newWidth, width, 'new width and old width are not the same');
```

Comment:
best way to find them is to remove the eslint-env comment at the top of the file and run `vjsstandard <file> | grep -v QUnit`